[generate_opcheck_tests] rename "success" to "xsuccess"

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #109641
* #109640
* #109639
* #109638
* __->__ #109637

Not BC breaking because no existing failures dict have "success" in
them.

Test Plan:
- new tests

zou3519 committed:
[generate_opcheck_tests] rename "success" to "xsuccess"

Not BC breaking because no existing failures dict have "success" in
them.

Test Plan:
- new tests

[ghstack-poisoned]

bdhirsh commented on torch/testing/_internal/optests/generate_tests.py:
```diff
@@ -193,7 +193,7 @@ def new_method(*args, **kwargs):
 
 
-TEST_OPTIONS = ("xfail", "skip", "success")
+TEST_OPTIONS = ("xfail", "skip", "xsuccess")
```
Just curious, is this just consistency with xfail? (and any reason not to do xskip?)

> Not BC breaking because no existing failures dict have "success" in
them.

also 🤣

soulitzer replied:
also try pronouncing it now lol

zou3519 replied:
Skip's a little different:

"xfail" = expected to fail
"xsuccess" = expected to succeed
"skip" = please skip this test

zou3519 committed:
Update on "[generate_opcheck_tests] rename "success" to "xsuccess""


Not BC breaking because no existing failures dict have "success" in
them.

Test Plan:
- new tests

[ghstack-poisoned]
